sdk/python/kfp/cli/compile_.py

```diff
@@ -155,13 +210,38 @@ def compile_(
         python_file=py, function_name=function_name)
     parsed_parameters = parse_parameters(parameters=pipeline_parameters)
     package_path = os.path.join(os.getcwd(), output)
+
+    manifest_options_provided = any([
```

Comment:
This can be removed now that `kubernetes_manifest_format` is available.

Reply:
@mprahl  The check for manifest_options_provided was added to provide a warning if someone supplies manifest-related options without --kubernetes-manifest-format.

Reply:
Oh yes, I see that now. Thanks!